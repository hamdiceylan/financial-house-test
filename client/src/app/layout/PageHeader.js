import React from 'react'
import { Icon, Header } from 'semantic-ui-react'

const PageHeader = ({icon, headerMessage, SubheaderMessage}) => {
  return (
    <Header as='h2'>
        <Icon name={icon} />
        <Header.Content>
            {headerMessage}
            <Header.Subheader>{SubheaderMessage}</Header.Subheader>
        </Header.Content>
    </Header>
  )
}

export default PageHeader
