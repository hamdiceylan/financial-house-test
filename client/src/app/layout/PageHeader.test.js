import React from 'react'
import { shallow } from 'enzyme'
import PageHeader from './PageHeader';

describe('PageHeader', () =>{
    const pageHeader = shallow(<PageHeader />);

    it('renders correctly', () => {
        expect(pageHeader).toMatchSnapshot();
    })
})

