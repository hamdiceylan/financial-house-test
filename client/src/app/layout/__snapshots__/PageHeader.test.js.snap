// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader renders correctly 1`] = `ShallowWrapper {}`;
